staging/src/k8s.io/apiserver/pkg/server/options/encryptionconfig/config.go

```diff
@@ -763,6 +732,56 @@ func kmsPrefixTransformer(ctx context.Context, config *apiserverconfig.KMSConfig
 	}
 }
 
+func primeAndProbeKMSv2(ctx context.Context, probe *kmsv2PluginProbe, kmsName string) {
```

Comment:
allow me if it is a dumb question. wondering if `k8s.io/client-go/util/workqueue` can be used to do this probe?

Reply:
This was on purpose - I wanted to both keep the logic simple and have a consistent interval for the checks (i.e. no exponential backoff).